docs/user/system_upgrades.md

```diff
@@ -0,0 +1,92 @@
+# Upgrading your system
+
+Brewblox is designed to be compatible with the majority of Linux-based systems.
+but there still are dependencies, and those have their own End of Life dates.
```

Comment:
"there are still dependencies" -> but we require two main dependencies on the host: docker and python.